substrate/frame/recovery/Cargo.toml

```
@@ -17,41 +17,25 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { features = ["derive"], workspace = true }
-frame-benchmarking = { optional = true, workspace = true }
-frame-support = { workspace = true }
-frame-system = { workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"] }
```

```suggestion
frame = { workspace = true, features = ["runtime"] }
```

We are not using the `experimental` flag anymore for the umbrella crate to avoid propagating this flag to the whole codebase

---

Got it, will update thanks!